webpack.mix.js

```diff
@@ -6,6 +6,8 @@ let purgeCssOptions = {
   enabled: true,
   // List of regex of CSS class to not remove
   whitelistPatterns: [/^ball-pulse/],
+  whitelistPatterns: [/^vs__/],
+  whitelistPatternsChildren: [/^vs-/],
   // List of regex of CSS class name whose child path CSS class will not be removed
   //  ex: to exclude "jane" in "mary jane": add "mary")
   whitelistPatternsChildren: []
```

Comment:
Please fix it like this:
```suggestion
  // List of regex of CSS class name whose child path CSS class will not be removed
  //  ex: to exclude "jane" in "mary jane": add "mary")
  whitelistPatternsChildren: [/^vs-/],
```